go.mod

```diff
@@ -42,6 +42,7 @@ require (
 	golang.org/x/net v0.0.0-20200520004742-59133d7f0dd7
 	golang.org/x/sys v0.0.0-20210511113859-b0526f3d8744
 	golang.org/x/text v0.3.6 // indirect
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
How do you feel about converging on `ghodss/yaml`? I chose it for `create` as that's the yaml library used in dd-go

Reply:
I think this branch needs a `go mod tidy`

Reply:
Not too familiar with this, but when I ran it, no changes were registered and a force push does nothing.

Reply:
Seems gone now? /shrug